src/serialize.rs

```diff
@@ -41,7 +45,7 @@ fn encode_size(f: &mut dyn Write, size: u64) -> std::io::Result<()> {
             ((size) & 0xff) as u8,
         ])?;
     } else {
-        panic!("atom size exceeded maximum {}", size);
```

Comment:
this is somewhat unrelated, but we shouldn't have `panic!()`s. It's better to fail the clvm program than to terminate the process.